src/coreclr/nativeaot/Runtime/startup.cpp

```diff
@@ -177,15 +177,15 @@ static bool InitDLL(HANDLE hPalInstance)
 
 bool DetectCPUFeatures()
 {
-#if defined(HOST_X86) || defined(HOST_AMD64) || defined(HOST_ARM64)
+#if defined(HOST_X86) || defined(HOST_AMD64) || defined(HOST_ARM64) || defined(HOST_LOONGARCH64)
     g_cpuFeatures = minipal_getcpufeatures();
 
     if ((g_cpuFeatures & g_requiredCpuFeatures) != g_requiredCpuFeatures)
     {
         PalPrintFatalError("\nThe required instruction sets are not supported by the current CPU.\n");
         RhFailFast();
     }
-#endif // HOST_X86|| HOST_AMD64 || HOST_ARM64
+#endif // HOST_X86|| HOST_AMD64 || HOST_ARM64 || HOST_LOONGARCH64
```

Comment:
```suggestion
#endif // HOST_X86|| HOST_AMD64 || HOST_ARM64
```

Reply:
Got it.
I've reverted these two files.
Thanks for your guidance!